demos/layout-grid.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
```

Comment:
Actually, removing the warning I mentioned, as the problem with MS Edge misreporting custom property support seems to be exclusive to the preview version of Edge.